recipes/fpsim2/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "fpsim2" %}
+{% set version = "0.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/chembl/FPSim2/archive/{{ version }}.tar.gz
+  sha256: 1312ea16c421c1e47f2dc9b35c5c713f5878ed0fd2e8eff362b45b8cf9954fd3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - setuptools
+    - {{ pin_compatible('numpy') }}
```

Comment:
This is backwards, see https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy